.travis.yml

```diff
@@ -36,6 +36,7 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
+  - pip freeze
```

Comment:
minor: Out of curiosity, why is that needed?

Reply:
It's useful for debugging Travis in general: it lists the packages that are installed. 

Here for instance, I was able to see that Black 20.8b1 was installed on Travis while I am running Back 19.10b0 locally which then explained why the Black tests worked locally but not on Travis. 20.8b1 changed how it deals with trailing commas...

Reply:
Ah ok I see! I knew the command but I didn't think that use case :P Indeed is really useful ;)